src/main/java/org/craftercms/studio/api/v2/dal/DependencyDAO.java

```diff
@@ -55,4 +55,20 @@
      * @return List of hard dependencies
      */
     List<Map<String, String>> getHardDependenciesForList(Map params);
+
+    /**
+     * Get items depending on given paths
+     *
+     * @param params SQL query parameters
+     * @return List of items depending on given paths
+     */
+    List<String> getDependentItems(Map params);
```

Comment:
I think it would be a good idea to start using the `@Param` annotation with the actual parameters instead of using a Map for all methods because it helps in reading the code & documenting

Reply:
@dejan-brkic let's discuss and see if possible.